### ISSUE TITLE
Tibialis anterior tendon rupture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11170]
**Preferred term label:**
Tibialis anterior tendon rupture

**Synonyms**

**Definition (free text, please give PubMed ID)**
Leg/ankle/foot MRI or ultrasound showing partial vs complete rupture of the tibialis anterior tendon

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Anterior tibialis tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001275
- label: Tibialis anterior tendon rupture
- parents: [Abnormal ankle morphology (HP:0034673)](https://hpo.jax.org/app/browse/term/HP:0034673)
- pmid: PMID:31501017
- def: Partial or complete breakage (rupture) of the tibialis anterior tendon (TAT). The TAT is the principal dorsiflexor of the ankle joint and secondary invertor of the foot. It has important roles in the different stages of gait; in the swing phase, it concentrically dorsiflex the ankle for foot clearance and during plantarflexion of the ankle in the stance phase, it eccentrically contract to stabilize the ankle and prevent damage to the foot and ankle. Clinically, presentations in acute lacerations or direct trauma cases are straight forward, in chronic and atraumatic cases usually the symptoms at the time of TAT rupture are more subtle. Other manifestations include mild pain, localized mass, palpable gap, loss of strength, instability during gait, gait disturbances with forefoot dropping and stumbling and high stepping gait.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11170
- synonyms:
